lib/updox/models/model.rb

```diff
@@ -7,30 +7,39 @@ class Model < Hashie::Trash
       include Hashie::Extensions::IgnoreUndeclared
       include Hashie::Extensions::IndifferentAccess
 
+      attr_writer :updox_status
+
       LIST_TYPE = 'undefined'
       LIST_NAME = 'models'
       ITEM_TYPE = 'model'
 
       property :item, required: false
       property :items, required: false
-      property :updox_status, default: {}
 
       def successful?
-        self[:updox_status].dig('successful')
+        @updox_status['successful']
```

Comment:
How do these work? Is `updox_status` a hash here? It looks like these methods are just trying to access something, but I don't see the writer method inputing values anywhere. What is the value is being written here?

Reply:
Supposed to be a hash - I guess the issue is that Updox _always_ returns success even on failure and then you have to parse the response to get whether it _actually_ failed so they kind of violate the idea of having an HTTP status (example https://github.com/WeInfuse/updox/blob/master/test/samples/calendars_list.response.json) and so this was a helper I added so I could just do `successful?` and not have to dig around in the json.